Dockerfile

```diff
@@ -1,10 +1,31 @@
 FROM circleci/node:10.11
 
-WORKDIR /opt/sinopia_editor/
+# Allow build-time arguments (for, e.g., docker-compose)
```

Comment:
This is the mechanism that allows us to inject environment variables at docker build time. `ARG` is for build arguments; `ENV` is for runtime arguments. `ENV` values are of limited utility in this case, since they are **not** available to the editor application via `process.env` (because `process.env` is a node thing, not a browser thing).